packages/@okta/vuepress-site/docs/guides/set-up-event-hook/overview/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Overview
+---
+
+Event Hooks are outbound calls from Okta that can be used to notify your own software systems of events occurring in your Okta org. See [Event Hooks](/docs/concepts/event-hooks/) for an overview. This guide provides the steps for setting up an Event Hook in your Okta org, which are:
+
+1. Implement your external web service to receive Event Hook calls from Okta.
```

Comment:
Are these numbers all expected to be `1.`?

Reply:
Yes, it's just markdown notation for a numbered list. Will get rendered as sequence of numbers.